zabbixsender/zabbixsvc/zabbixsvc.go

```diff
@@ -80,15 +128,15 @@ func (h *JSONHandler) HandlePost(w http.ResponseWriter, r *http.Request) {
 		log.Debugf("sending zabbix metrics, host: '%s' key: '%s', value: '%s'", host, key, value)
 	}
 
-	res, err := h.zabbixSend(metrics)
+	res, err := h.zabbixSend(metrics, ammountOfResolved)
```

Comment:
Why are you passing metric here,  `sender` shouldn't be doing this just loop thru `alerts` check whether it's resolved and `increment`